src/execution_tree/primitives/annotate_primitive.cpp

```diff
@@ -108,7 +108,8 @@ namespace phylanx { namespace execution_tree { namespace primitives {
             annotation tmp(args);
```

Comment:
Why do we need this copy of `args`?

Reply:
Good catch, it would appear that we don't.